LV_analysis/1_get_multiplier_LV_coverage_Pa.py

```diff
@@ -0,0 +1,335 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:generic_expression] *
+#     language: python
+#     name: conda-env-generic_expression-py
+# ---
+
+# # Coverage of MultiPLIER LV
```

Comment:
Do you want to note here that this script is for Pseudomonas data? Looks like this intro is just copy-pasted from the human version of the script (which is fine if that's what you want).